### PR TITLE
Update message size test 

### PR DESCRIPTION
**Pull request by MarwanMashaly1**
Updated the Mavlink dialect to have different messages based on the data type therefore updating the message size test to be based on the data type.

### CODE REVIEW

File: libraries/messages/src/lib.rs [hunk 1]
```diff
@@ -70,16 +70,33 @@ impl Message {
...
         fn message_size(msg: Message) {
             let bytes = postcard::to_allocvec(&msg).unwrap();
 
-            dbg!(msg);
-            assert!(dbg!(bytes.len()) <= MAX_SIZE);
+            dbg!(&msg);
+            // The size of the message should be less than or equal the maximum size.
+            match msg.data {
+                crate::Data::State(_) => {
+                    assert!(dbg!(bytes.len()) <= 13);
```
Comment (NoahSprenger) [inline libraries/messages/src/lib.rs:85]: Can you define these to be constants like MAX_SIZE was.

**Review by NoahSprenger — CHANGES_REQUESTED** [review]

File: libraries/messages/src/lib.rs [hunk 2]
```diff
@@ -24,7 +24,10 @@ pub mod sensor;
 pub mod sensor_status;
 pub mod state;
 
-pub const MAX_SIZE: usize = 64;
+pub const MAX_HEALTH_SIZE: usize = 47;
+pub const MAX_SENSOR_SIZE: usize = 53;
+pub const MAX_STATE_SIZE: usize = 13;
+pub const MAX_LOGNCOMMAND_SIZE: usize = 15;
```
Comment (NoahSprenger) [inline libraries/messages/src/lib.rs:30]: I know they are the same size, but they might not always be the same size, can you split these.

**Review by NoahSprenger — CHANGES_REQUESTED** [review]

File: libraries/messages/src/lib.rs [hunk 3]
```diff
@@ -24,7 +24,10 @@ pub mod sensor;
 pub mod sensor_status;
 pub mod state;
 
-pub const MAX_SIZE: usize = 64;
```
Comment (NoahSprenger) [inline libraries/messages/src/lib.rs:27]: Also, please keep this.

Comment (MarwanMashaly1) [reply libraries/messages/src/lib.rs:27]: sure but it is not used anymore though

**Review by NoahSprenger — APPROVED** [review]

**Pull request merged**